SQL/New_patches/2018-09-07_EEG_tables.sql

```diff
@@ -0,0 +1,259 @@
+  CONSTRAINT `FK_physiological_output_type_PhysiologicalOutputTypeID`
+    FOREIGN KEY (`PhysiologicalOutputTypeID`)
+    REFERENCES `physiological_output_type` (`PhysiologicalOutputTypeID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+
+
+-- Create a physiological_parameter_file table that will store all JSON
+-- information that accompanies the BIDS physiological dataset
+CREATE TABLE `physiological_parameter_file` (
+  `PhysiologicalParameterFileID` INT(10) UNSIGNED NOT NULL AUTO_INCREMENT,
+  `PhysiologicalFileID`          INT(10) UNSIGNED NOT NULL,
+  `ParameterTypeID`              INT(10) UNSIGNED NOT NULL,
+  `InsertTime`                   INT(10) UNSIGNED NOT NULL,
+  `Value`                        VARCHAR(255),
+  PRIMARY KEY (`PhysiologicalParameterFileID`),
+  CONSTRAINT `FK_physiological_file_PhysiologicalFileID`
+    FOREIGN KEY (`PhysiologicalFileID`)
+    REFERENCES `physiological_file` (`PhysiologicalFileID`)
+    ON DELETE CASCADE,
+  CONSTRAINT `FK_parameter_type_ParameterTypeID`
+    FOREIGN KEY (`ParameterTypeID`)
+    REFERENCES `parameter_type` (`ParameterTypeID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+
+
+-- Create a physiological_status_type table
+CREATE TABLE `physiological_status_type` (
+  `PhysiologicalStatusTypeID` TINYINT(2) UNSIGNED NOT NULL AUTO_INCREMENT,
+  `ChannelStatus`             VARCHAR(10)         NOT NULL,
+  PRIMARY KEY (`PhysiologicalStatusTypeID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+
+
+-- Create a physiological_channel_type table
+CREATE TABLE `physiological_channel_type` (
+  `PhysiologicalChannelTypeID` TINYINT(3) UNSIGNED NOT NULL      AUTO_INCREMENT,
+  `ChannelType`                VARCHAR(255)        NOT NULL,
+  `ChannelDescription`         VARCHAR(255)        DEFAULT NULL,
+  PRIMARY KEY (`PhysiologicalChannelTypeID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+
+
+-- Create a physiological_channel table that will store all information from
+CREATE TABLE `physiological_channel` (
+  `PhysiologicalChannelID`     INT(10) UNSIGNED    NOT NULL      AUTO_INCREMENT,
+  `PhysiologicalFileID`        INT(10) UNSIGNED    NOT NULL,
+  `PhysiologicalChannelTypeID` TINYINT(3) UNSIGNED NOT NULL,
+  `PhysiologicalStatusTypeID`  TINYINT(2) UNSIGNED DEFAULT NULL,
+  `Name`                       VARCHAR(50)         NOT NULL,
+  `Description`                VARCHAR(255)        DEFAULT NULL,
+  `SamplingFrequency`          INT(6)              DEFAULT NULL,
+  `LowCutoff`                  DECIMAL(8,3)        DEFAULT NULL,
+  `HighCutoff`                 DECIMAL(8,3)        DEFAULT NULL,
+  `ManualFlag`                 DECIMAL(5,4)        DEFAULT NULL,
+  `Notch`                      INT(6)              DEFAULT NULL,
+  `StatusDescription`          VARCHAR(255)        DEFAULT NULL,
+  `Unit`                       VARCHAR(255)        DEFAULT NULL,
+  `File`                       VARCHAR(255)        DEFAULT NULL,
+  `SoftwareFilters`            VARCHAR(255)        DEFAULT NULL,
```

Comment:
What's the format of this? By the name of it it sounds like it should be normalized into a different table.